hlapipeline/alignimages.py

```diff
@@ -251,9 +272,8 @@ def perform_align(input_list, archive=False, clobber=False, update_hdr_wcs=False
 
         if total_num_sources < MIN_OBSERVABLE_THRESHOLD:
             print("Not enough sources ({}) found in image {}".format(total_num_sources,imgname))
-            filteredTable[index]['status'] = 1
+            filteredTable[index]['status'] = 1 #TODO: Should this update the status of all images in filteredTable with the value 1? What about processMsg?
```

Comment:
For the current set of inputs, this would be fine.  We can review what should be done for different sets of inputs where some images may not have any source and others do.  Although, future use of 'expand_refcat' in tweakwcs will also mitigate this as well.

Reply:
Resolved by commit [a85754b](https://github.com/spacetelescope/hlapipeline/pull/73/commits/a85754bbb0f58f4db3b406be9ecf119757052f60)